src/applications/disability-calculator/utils/helpers.js

```diff
@@ -38,5 +38,20 @@ export function calculateRating(arr) {
   }
   return [undefined, undefined];
 }
-// will return array with two elements. first element in array is rounded rating
+// calculateRating will return array with two elements. first element in array is rounded rating
 // and second element is the actual rating
+
+// this is a validator to check if there are at least two ratings
+export function shouldCalculate(ratingsArr) {
+  const checkIfTwo = ratingsArr;
+  const returnArr = [];
+  checkIfTwo.forEach(e => {
+    if (typeof e === 'number') {
+      returnArr.push(e);
+    }
+  });
+  if (returnArr.length >= 2) {
+    return true;
+  }
+  return false;
+}
```

Comment:
I think these 2 helper functions might not be necessary.  You might want to just disable the "Calculate" button if `ratings.length < 2`, that way you don't even need to perform these checks.

Reply:
Great idea of disabling the button, but this won't work because `ratings.length` will always be at least `2` from the `defaultRatings` 

```
const defaultRatings = [
  {
    rating: '',
    description: '',
  },
  {
    rating: '',
    description: '',
  },
];
```
 
I'll have to utilize the helper function `shouldCalculate` to see if rating values are not empty.

Reply:
These `defaultRatings` are only used when initializing the app, or if the user hits reset.  In other words, `state.ratings` is its own, separate array from `defaultRatings` and only creates a copy of `defaultRatings` on load or reset, so it can have less than 2 items.